py/server/deephaven/experimental/iceberg.py

```diff
@@ -813,6 +813,9 @@ def create_table(self, table_identifier: str, table_definition: TableDefinitionL
         return IcebergTableAdapter(self.j_object.createTable(table_identifier,
                                                              TableDefinition(table_definition).j_table_definition))
 
+    def close(self):
+        return self.j_object.close()
+
```

Comment:
This is all good. But we may want to make `IcebergTableAdapter` a Python ContextManager which conceptually is the equivalent to Java's AutoClosable.  I'd also like to see a test case if possible but would not require it since I know it is not easiest to set up the environment.